src/main/resources/view/StallListCard.fxml

```diff
@@ -7,7 +7,7 @@
 <?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.layout.VBox?>
 
-<HBox id="cardPane" fx:id="cardPane" prefHeight="96.0" prefWidth="350.0" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+<HBox id="cardPane" fx:id="cardPane" prefHeight="96.0" prefWidth="350.0" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
I see this change - I'm not sure why is this changed to JavaFX 8.

Reply:
This is for standardization purposes, as all other FXML in the existing project uses JavaFX 8.